anaconda_packaging_utils/api/github_api.py

```diff
@@ -46,25 +46,25 @@ def __init__(self, message: str):
 class GitHubApi:
     """
     Singleton wrapper to the PyGithub project. This "ensures" that we only construct and authenticate the underlying
-    Github object once. All convenience member.
+    Github object once.
 
     As best as I can tell, the `PyGitHub` project is not guaranteed to be thread safe. However, read-only GitHub API
     requests are not likely to cause a threading issue, by their very nature.
     """
 
     # The GitHub API is wrapped in a list as a cheesy way to work around an initialization problem. Defaulting to `None`
-    # or using an `Optional` causes the static analyzer to freak out on every use of the `_gh`, even if the static
+    # or using an `Optional` causes the static analyzer to freak out on every use of the `__gh`, even if the static
     # variable has to have been initialized by instance-method call time.
-    _gh: list[Github] = []
+    __gh: list[Github] = []
 
     def __init__(self) -> None:
         """
         Constructs a GitHubApi Instance
         :raises ApiException: If there was a failure to authenticate.
         """
-        if len(GitHubApi._gh) == 0:
+        if len(GitHubApi.__gh) == 0:
```

Comment:
I think this could be more efficient
```suggestion
        if not GitHubApi.__gh:
```

Reply:
Unlike C++'s `size()`, `len` is usually O(1): https://wiki.python.org/moin/TimeComplexity
But this reads better anyways. Sometimes I over-specify in python to ensure I get a boolean expression and not something unwanted.